ifsbench/config_mixin.py

```diff
@@ -0,0 +1,36 @@
+from typing import Dict, List, Union
+
+from pydantic import BaseModel, model_validator
+from typing_extensions import Self
+
+__all__ = ['ConfigMixin', 'CLASSNAME', 'RESERVED_NAMES']
+
+CLASSNAME = 'classname'
+RESERVED_NAMES = [
+    CLASSNAME,
+]
+
+
+class ConfigMixin(BaseModel):
+
+    @classmethod
+    def from_config(
```

Comment:
Same here, all outward-facing API should have a docstring, I think.

Reply:
Done